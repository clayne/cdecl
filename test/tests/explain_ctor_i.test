cdecl @ /dev/null @ -xc++ @ explain C::C(int) @ 0
