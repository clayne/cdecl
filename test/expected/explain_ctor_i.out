declare C as constructor (int)
